c/index.md

```diff
@@ -0,0 +1,10 @@
+# List of the artists starting with C
+
+- [Claudiot](https://claudiot.md)
```

Comment:
This looks good, you won't need the `https://` scheme in the URL though, just `claudiot.md`